packages/block-library/src/image/edit.native.js

```diff
@@ -327,8 +327,10 @@ class ImageEdit extends React.Component {
 		const getSizeOptions = () => (
 			<Picker
 				hideCancelButton={ true }
+				title={ __( 'Image Size' ) }
 				ref={ ( instance ) => picker = instance }
 				options={ getSizeOptionsItems() }
+				leftAlign={ true }
```

Comment:
`<Picker ... leftAlign />` is valid as well

Reply:
Sorry misunderstood you here earlier. I saw the second comment, will update in the next commit.